ga4gh/cli.py

```diff
@@ -936,6 +956,7 @@ def client_main(parser=None):
     addCallsetsGetParser(subparsers)
     addVariantsGetParser(subparsers)
     addReferencesBasesListParser(subparsers)
+    addDatasetsGetParser(subparsers)
```

Comment:
switch this line with the line above, so we have all the add*GetParser calls together, just for clarity